src/tool/yamlupgrade.cpp

```diff
@@ -508,6 +508,86 @@ static bool upgrade_item_group_db( std::string file, const uint32 source_version
 			body << YAML::EndMap;
 		}
 
+		if( source_version == 3 ){
```

Comment:
Not sure what happens when someone with < version 3 upgrades and the upgrades above are also done at the same time. Could cause problems...